v2/contribute/decisions/reactsdk/0001-move-override-config-to-separate-component-prop.mdx

```diff
@@ -0,0 +1,46 @@
+---
+id: "0001"
+title: Move override config to separate component's prop
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Move override config to separate component's prop
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-21" created="2022-12-21" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["rishabhpoddar"]} />
+
+## Context and Problem Statement
+
+We may want to separate override config from supertokens-auth-react init, so user does have to call `init` both for `web-js` and `auth-react`. See related [issue](https://github.com/supertokens/supertokens-auth-react/issues/616)
```

Comment:
This reasoning is incorrect. We want to extract the override config for components only (and not functions) so that the auth-react SDK can be initialised without having to depend on react.

Reply:
Note that the functions override remains in the auth-react init as it is right now.